webpack/components/NewTemplateSync/components/NewTemplateSyncForm/NewTemplateSyncFormHelpers.js

```diff
@@ -24,14 +29,30 @@ export const syncFormSchema = (syncType, settingsObj, validationData) => {
           repo: Yup.string()
             .test(
               'repo-format',
-              `Invalid repo format, must start with one of: ${validationData.repo.join(
-                ', '
-              )}`,
+              `${__(
+                'Invalid repo format, must start with one of: '
+              )}${validationData.repo.join(', ')}`,
               repoFormat(validationData.repo)
             )
             .required("can't be blank"),
         };
       }
+      if (setting.name === 'http_proxy_policy') {
+        return {
+          ...memo,
+          http_proxy_policy: Yup.mixed().test(
+            'http-proxy-available',
+            __(
+              'No HTTP proxies available. Please select a different policy or verify that you have selected the correct organization and location.'
+            ),
```

Comment:
This is only visible after clicking "No HTTP proxies available" and also blocks the user form changing from import to export. I think it might be better to not show the option for a custom http proxy is the user doesnt have any (or, more complicated I think, to show the warning and only block submit)